concent_api/core/tests/test_unit_custom_manager_queryset.py

```diff
@@ -0,0 +1,167 @@
+        ('concent_upload_rate', 'download_leadin_time', 'concent_messaging_time', 'custom_protocol_times'), [
+            (48, 3, 2, True),
+            (96, 6, 4, True),
+            (192, 12, 8, True),
+            (384, 24, 16, True),
+        ])
+    def test_that_maximum_download_time_query_gives_correct_value(
+            self,
+            concent_upload_rate,
+            download_leadin_time,
+            concent_messaging_time,
+            custom_protocol_times,
+    ):
+        with override_settings(
+            CONCENT_UPLOAD_RATE=concent_upload_rate,
+            DOWNLOAD_LEADIN_TIME=download_leadin_time,
+            CONCENT_MESSAGING_TIME=concent_messaging_time,
+            CUSTOM_PROTOCOL_TIMES=custom_protocol_times,
+        ):
+            maximum_download_deadline = calculate_maximum_download_time(
+                size=self.report_computed_task.size,
+                rate=settings.CONCENT_UPLOAD_RATE,
+            )
+            assert_that(
+                Subtask.objects_with_timing_columns.
+                get(subtask_id=self.report_computed_task.task_to_compute.subtask_id).maximum_download_time
+            ).is_equal_to(
+                maximum_download_deadline
+            )
+
+    @pytest.mark.django_db
+    def test_that_maximum_download_time_query_gives_correct_value_without_custom_protocol_times(self):
+        with override_settings(
+                CUSTOM_PROTOCOL_TIMES=False,
+        ):
+            maximum_download_deadline = calculate_maximum_download_time(
+                size=self.report_computed_task.size,
+                rate=settings.CONCENT_UPLOAD_RATE,
+            )
+            assert_that(
+                Subtask.objects_with_timing_columns.
+                get(subtask_id=self.report_computed_task.task_to_compute.subtask_id).maximum_download_time
+            ).is_equal_to(
+                maximum_download_deadline
+            )
+
+    @pytest.mark.django_db
+    @pytest.mark.parametrize(
+        ('concent_upload_rate', 'download_leadin_time', 'concent_messaging_time', 'custom_protocol_times'), [
+            (48, 3, 2, True),
+            (96, 6, 4, True),
+            (192, 12, 8, True),
+            (384, 24, 16, True),
+        ])
+    def test_that_download_deadline_query_gives_correct_value(
+        self,
+        concent_upload_rate,
+        download_leadin_time,
+        concent_messaging_time,
+        custom_protocol_times
+    ):
+        with override_settings(
+            CONCENT_UPLOAD_RATE=concent_upload_rate,
+            DOWNLOAD_LEADIN_TIME=download_leadin_time,
+            CONCENT_MESSAGING_TIME=concent_messaging_time,
+            CUSTOM_PROTOCOL_TIMES=custom_protocol_times,
+        ):
+            assert_that(
+                Subtask.objects_with_timing_columns.
+                get(subtask_id=self.report_computed_task.task_to_compute.subtask_id).download_deadline
+            ).is_equal_to(
+                (Subtask.objects_with_timing_columns.
```

Comment:
You still have this unnecessary pair of parentheses here. There's already a parenthesis in `.is_equal_to(`. You don't need to do `.is_equal_to((Subtask ... ))`.

Reply:
Done.